app/service/send_notification.py

```diff
@@ -116,6 +117,8 @@ def get_reply_to_text(notification_type, sender_id, service, template):
                 message = 'SMS sender not found'
                 reply_to = dao_get_service_sms_sender_by_id(service.id, sender_id).get_reply_to_text()
         except NoResultFound:
+            if len(message) == 0:
+                message = 'No Results Found'
```

Comment:
```suggestion
```

Reply:
The try block needs an else statement. Something like
```
else:
    message = f'Invalid notification type: {notification_type}'
```

All other routes lead to message being set.